src/vs/workbench/contrib/audioCues/browser/audioCueService.ts

```diff
@@ -165,6 +165,12 @@ export class AudioCue {
 		settingsKey: 'audioCues.debuggerStoppedOnBreakpoint',
 	});
 
+	public static readonly noInlayHints = AudioCue.register({
+		name: localize('audioClues.noInlayHints', 'Line has no Inlay Hints'),
```

Comment:
This is being read out as is in the audio cue help.
My first thought was that this is being played for every line that has no audio cue.

I suggest to change the description to "No Inlay Hints available for the current line" to indicate that it is different than the other audio cues.

Reply:
Makes sense